angelspetrochem/prototypes/technology/petrochem-petro-chemistry.lua

```diff
@@ -64,14 +64,6 @@ data:extend({
         type = "unlock-recipe",
         recipe = "condensates-oil-refining",
       },
-      {
-        type = "unlock-recipe",
-        recipe = "solid-fuel-naphtha",
-      },
-      {
-        type = "unlock-recipe",
-        recipe = "solid-fuel-fuel-oil",
-      },
```

Comment:
Why are these deleted? Seems like in `angelspetrochem/prototypes/global-override/bobrevamp.lua`, these are moved to flammables?

```
  OV.remove_unlock("angels-oil-processing", "solid-fuel-fuel-oil")
  OV.remove_unlock("angels-oil-processing", "solid-fuel-naphtha")

  OV.add_unlock("flammables", "solid-fuel-fuel-oil")
  OV.add_unlock("flammables", "solid-fuel-naphtha")
```

Either the OV.remove_unlock is also obsolete (which tech does unlock them?)
Or the recipes should unlock by this tech by default?